.github/actions/check-issues-are-disabled/action.yaml

```diff
@@ -0,0 +1,25 @@
+name: Check Issues Are Disabled
+description: Checks whether the current repository has disabled the issues feature.
+
+outputs:
+  issues-disabled:
+    description: Whether the issues are disabled or not.
+    value: ${{ steps.repository-has-disabled-issues.outputs.issues-disabled }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup Scripts
+      shell: pwsh
+      run: |
+        "${{ github.action_path }}" >> $Env:GITHUB_PATH
+        (Resolve-Path "${{ github.action_path }}/../../../Scripts").Path >> $Env:GITHUB_PATH
+  
+    - name: Check Issues Are Disabled In Current Repository
+      shell: pwsh
+      id: repository-has-disabled-issues
+      run: |
+        $issueResponse = $(gh issue list --repo '${{ github.repository }}')
+        $message = "the '${{ github.repository }}' repository has disabled issues"
+        $issuesAreDisabled = ("$issueResponse" -match "$message")
```

Comment:
Are the parenthesis and double quotes necessary here?
`$issuesAreDisabled = $issueResponse -match $message` should have the same effect (if `$issueResponse` is already a string at this point).

Reply:
I addressed this and made it more general, but `"$issueResponse"` needs to have double quotes as it is originally an `Object[]`.